src/ProjectOrigin.VerifiableEventStore/Services/EventStore/IEventStore.cs

```diff
@@ -4,7 +4,15 @@ namespace ProjectOrigin.VerifiableEventStore.Services.EventStore;
 
 public interface IEventStore
 {
-    Task StoreBatch(Batch batch);
+    Task Store(VerifiableEvent @event);
+
     Task<Batch?> GetBatch(EventId eventId);
```

Comment:
As we discussed, should always return a batch or throw exception, then add status if finalized.